engine/src/main/scala/cromwell/services/package.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell
+
+import java.sql.Timestamp
+
+import cromwell.backend.BackendJobDescriptorKey
+import cromwell.core.WorkflowId
+import cromwell.engine.ExecutionIndex._
+
+
+package object services {
```

Comment:
I'd prefer to keep the package objects extremely slim. Perhaps put this elsewhere? Since this is scala and not java they could even just live as-is in a single file. 

I don't really care that much about this specific case though, so feel free to disregard. It's more of a broken windows thing at the moment.